### ISSUE TITLE
[Components and pattern standards] Your content or feature doesn't use standard global navigation components or is missing a standard navigation component.  (04.02.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#47165]
#### VFS team name
Digital Health Platform

#### VFS product name
Connect Your Device

#### VFS feature name
Device Connection Experience

#### Point of Contact/Reviewers
Jim Ryan (@jimuhh) - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your content or feature doesn't use standard global navigation components or is missing a standard navigation component. 

### Issue Details
Page is missing breadcrumb showing its place within site hierarchy.

### VA.gov Experience Standard
[Category Number 04, Issue Number 02](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
Please work w/ sitewide content and IA to determine a place for this page.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by shiragoodman** [comment]
@mnorthuis The [team communicated in the Staging Review ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/46594#issuecomment-1269028482) that they don't need to have breadcrumbs per the IA design. Can you confirm please? If so, we can close out this ticket and unassign the DHP team.

**Comment by mnorthuis** [comment]
The team is correct, and Kim linked to my IA recommendations in the staging ticket that state that - the documented IA recs should be referenced when doing midpoint and staging reviews. The design system also captures some scenarios where a breadcrumb is not needed.

**Comment by shiragoodman** [comment]
Thanks @mnorthuis. Closing this issue an unassigning it from Digital Health Platform. FYI @jimuhh @robbie1226
